pkgs/by-name/pr/prometheus-folder-size-exporter/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  cargo,
+  rustc,
+  fetchFromGitHub,
+  rustPlatform,
+  lib,
+}:
+rustPlatform.buildRustPackage {
+  pname = "prometheus_folder_size_exporter";
+  version = "0.5.1";
+
+  src = fetchFromGitHub {
+    owner = "MindFlavor";
+    repo = "prometheus_folder_size_exporter";
+    rev = "5800b20e86855f407d9c5db48ad6f966a5bd2df2";
+    hash = "sha256-lvn24NAKv4YU1iU0U7NBBQVr2BQM1wi5M6czj/7gUSk=";
+  };
+
+  cargoHash = "sha256-FuXlFQBcVGJkUqkJdiX37a9d4QpHYES04XVYusWrAJo=";
+
+  nativeBuildInputs = [
+    cargo
+    rustc
+    rustPlatform.cargoSetupHook
+  ];
+
+  buildInputs = [
+  ];
```

Comment:
`buildRustPackage` should do this already
```suggestion
```

Reply:
huh, thought I had tried it without it and ti didnt work.  
Works fine without as ye expected